test/setup/variables.tf

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "org_id" {
+  description = "The numeric organization id"
+}
+
+variable "folder_id" {
```

Comment:
This should be the unprefixed folder_id (ie. `368577169113` instead of `folders/368577169113`) for consistency. In particular, it looks like the outputs assume the latter.

Reply:
Good catch, outputs and CB trigger configuration have been updated. I missed that point cause for project factory we can pass `folder_id` in any format, so my tests passed and I decided it's the right way https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/core_project_factory/scripts/preconditions/preconditions.py#L171-L174.